src/Components/Components/src/Infrastructure/ComponentStatePersistenceManager.cs

```diff
@@ -2,58 +2,60 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Buffers;
 using System.Collections.Generic;
-using System.Collections.ObjectModel;
-using System.Linq;
-using System.Runtime.CompilerServices;
+using System.IO.Pipelines;
```

Comment:
If we're using new parts of the BCL, it will be necessary to check it doesn't regress the post-linked sizes. The runtime team is working super hard to shave off individual kilobytes, so we have to be careful not to drop in 5KB or so if it's avoidable.

Reply:
I suppose the whole "state persistence" feature will link out and have no impact by default, is that right?

Reply:
To further clarify this, there was a bunch of work done in https://github.com/dotnet/aspnetcore/issues/27782#issuecomment-804973106 to make sure that `System.IO.Pipelines` would get trimmed because it saves 20KB. Just need to make sure we don't bring it back by default without specific planning for that.

Reply:
This was just left-over code. It's gone now